File: sdkwork-backend-sdk-go/types/content_vote_vo.go
package types

// 内容投票数据传输对象
type ContentVoteVO struct {
	CreatedAt string `json:"createdAt"`
	UpdatedAt string `json:"updatedAt"`
	Id int `json:"id"`
	Uuid string `json:"uuid"`
	V int `json:"v"`
	UserId int `json:"userId"`
	ContentType string `json:"contentType"`
	Rating string `json:"rating"`
	ContentId int `json:"contentId"`
	Metadata ContentVoteMetadata `json:"metadata"`
}

File: sdkwork-backend-sdk-csharp/Models/ContentVoteVO.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Backend.Models
{
    public class ContentVoteVO
    {
        public string? CreatedAt { get; set; }
        public string? UpdatedAt { get; set; }
        public int? Id { get; set; }
        public string? Uuid { get; set; }
        public int? V { get; set; }
        public int? UserId { get; set; }
        public string? ContentType { get; set; }
        public string? Rating { get; set; }
        public int? ContentId { get; set; }
        public ContentVoteMetadata? Metadata { get; set; }
    }
}
